### DISCUSSION TITLE
Reverse alias command

### DISCUSSION THREAD

**Discussion by davidoskky** [neomutt/neomutt#4150] [Development]
I use khard to store all my contacts and sync them with cardav.
The query_command works perfectly and allows to retrieve addresses from the program.
It would be nice if this could be done for reverse alias: I would like all the mail senders in the index menu to have the name I set in khard, without having to create the alias for all of them.
It could be done by calling the program every time I open a mailbox or by storing somewhere the contacts in some file and querying the external program only the first time an address is encountered.

I assume you could just make one script to export all that info in alias format, but that would require maintenance and would have to be issued each time you modify a contact.

**Comment by gahr** [comment]
I wouldn't mind a `reverse_alias` command.
